subxt/src/metadata/metadata_type.rs

```diff
@@ -97,7 +97,10 @@ struct MetadataInner {
     cached_storage_hashes: HashCache,
 }
 
-/// Runtime metadata.
+/// Internal metadata representation over the runtime metadata.
+///
+/// This structure holds the ability to easily translate strings and indexes to
+/// underlying metadata types.
```

Comment:
```suggestion
/// A representation of the runtime metadata received from a node.
```

(No need to be too specific I think :))